src/js/grapl-cdk/lib/engagement.ts

```diff
@@ -51,7 +52,7 @@ export class EngagementEdge extends cdk.NestedStack {
     constructor(scope: cdk.Construct, id: string, props: EngagementEdgeProps) {
         super(scope, id);
 
-        const ux_bucket = s3.Bucket.fromBucketName(
+        const ux_bucket = GraplS3Bucket.fromBucketName(
```

Comment:
technically, any usage besides `new s3.Bucket` could remain as `s3.Bucket` - I just wanted to get everywhere into the habit of using GraplS3Bucket